software/controller/integration_tests/pinch_valve_test.h

```diff
@@ -18,7 +18,7 @@ static constexpr Duration Delay{milliseconds(1000)};
 
 void RunTest() {
   hal.Init();
-  PinchValve pinch_valve(MotorIndex);
+  PinchValve pinch_valve(MotorIndex, "any", " for whatever");
```

Comment:
Yes, it will create debug variables and yes, in case of integration tests, we don't really do anything with them.

We may completely change how we do hardware-in-the-loop tests in the future. It certainly is not hard to expose the calibration as public.